include/gsl/gsl-lite.hpp

```diff
@@ -240,7 +240,7 @@
     typename std::enable_if<VA, R>::type
 
 #define gsl_REQUIRES_A(VA) \
-    , typename std::enable_if<VA, void*>::type = nullptr
+    , typename std::enable_if<VA, int>::type = 0
```

Comment:
This is a potentially ABI-breaking change, but `gsl_REQUIRES_A()` is neither a documented feature of *gsl-lite* nor used in any class template argument lists, so I think we can take the risk.

Reply:
I agree, the gsl_REQUIRES macros are implementation details, they are not meant to be used by consumers.